src/main/java/io/supertokens/inmemorydb/queries/ActiveUsersQueries.java

```diff
@@ -0,0 +1,43 @@
+package io.supertokens.inmemorydb.queries;
+
+import java.sql.SQLException;
+
+import io.supertokens.inmemorydb.config.Config;
+import io.supertokens.pluginInterface.exceptions.StorageQueryException;
+import io.supertokens.inmemorydb.Start;
+
+import static io.supertokens.inmemorydb.QueryExecutorTemplate.execute;
+import static io.supertokens.inmemorydb.QueryExecutorTemplate.update;
+
+public class ActiveUsersQueries {
+    static String getQueryToCreateUserLastActiveTable(Start start) {
+        return "CREATE TABLE IF NOT EXISTS " + Config.getConfig(start).getUserLastActiveTable() + " ("
+                + "user_id CHAR(36),"
```

Comment:
this should be 128 since we will also be storing session's user ID and that's 128 right?

Reply:
Okay. I didn't know that. 

Fixed.